applications_tests/dem_3d/periodic_boundary_gmsh.prm

```diff
@@ -0,0 +1,100 @@
+# Listing of Parameters
+# ---------------------
+
+#---------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+    set time step                = 1.5e-6
```

Comment:
Why is the timestep the same as the end time? So you are only doing one iteration?